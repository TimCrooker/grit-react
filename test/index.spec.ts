import path from 'path'
import { getGenerator } from 'grit-cli'

const generator = path.join(__dirname, '..')

test('defaults', async () => {
	const grit = await getGenerator({
		generator,
		mock: true,
		answers: {
			name: 'test',
			pmRun: 'yarn',
			features: 'none',
		},
	})
	await grit.run()

	expect(await grit.getOutputFiles()).toMatchSnapshot()
})
