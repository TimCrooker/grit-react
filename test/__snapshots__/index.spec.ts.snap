// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`defaults 1`] = `
Array [
  ".dockerignore",
  ".eslintignore",
  ".gitignore",
  "Dockerfile",
  "README.MD",
  "_.eslintrc.js",
  "__tests__/index.spec.ts",
  "_tsconfig.json",
  "jest.config.ts",
  "package.json",
  "public/icons/github-icon.svg",
  "public/icons/linkedin-icon.svg",
  "public/icons/pankod-icon.svg",
  "public/icons/react-icon.svg",
  "public/icons/twitter-icon.svg",
  "public/icons/youtube-icon.svg",
  "src/App.tsx",
  "src/components/css/button/index.css",
  "src/components/css/button/index.spec.tsx",
  "src/components/css/button/index.stories.tsx",
  "src/components/css/button/index.tsx",
  "src/components/css/card/index.css",
  "src/components/css/card/index.spec.tsx",
  "src/components/css/card/index.stories.tsx",
  "src/components/css/card/index.tsx",
  "src/components/css/cards/index.css",
  "src/components/css/cards/index.spec.tsx",
  "src/components/css/cards/index.tsx",
  "src/components/css/footer/index.css",
  "src/components/css/footer/index.spec.tsx",
  "src/components/css/footer/index.tsx",
  "src/components/css/header/index.css",
  "src/components/css/header/index.spec.tsx",
  "src/components/css/header/index.tsx",
  "src/components/css/index.ts",
  "src/components/css/logo/index.spec.tsx",
  "src/components/css/logo/index.tsx",
  "src/components/css/main/index.css",
  "src/components/css/main/index.spec.tsx",
  "src/components/css/main/index.tsx",
  "src/index.html",
  "src/index.tsx",
  "src/meta.json",
  "src/pages/index.tsx",
  "src/styles/global.css",
  "webpack.common.js",
  "webpack.dev.js",
  "webpack.prod.js",
]
`;
